CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `MimirHPANeedsToBeScaledUp` alert, to detect when Mimir's HPAs have reached maximum capacity.
```

Comment:
Can we align names with https://github.com/giantswarm/prometheus-rules/blob/5276895723d59b94b7a083b80a2ced0d93be76e6/helm/prometheus-rules/templates/platform/atlas/alerting-rules/loki.rules.yml#L111 ?

Reply:
Maybe the alert should be the same?

Reply:
Renamed it to `MimirHPAReachedMaxReplicas`